protected/humhub/modules/comment/notifications/CommentDeleted.php

```diff
@@ -0,0 +1,73 @@
+<?php
+
+/**
+ * @link https://www.humhub.org/
+ * @copyright Copyright (c) 2017 HumHub GmbH & Co. KG
+ * @license https://www.humhub.com/licences
+ */
+
+namespace humhub\modules\comment\notifications;
+
+use humhub\modules\comment\models\Comment;
+use humhub\modules\content\components\ContentActiveRecord;
+use humhub\modules\notification\components\BaseNotification;
+use humhub\modules\notification\models\Notification;
+use humhub\modules\user\models\User;
+use humhub\modules\user\notifications\Mentioned;
+use Yii;
+use yii\bootstrap\Html;
+
+/**
+ * CommentDeletedNotification is fired when admin deletes a comment
+ */
+class CommentDeleted extends BaseNotification
+{
+    /**
+     * @inheritdoc
+     */
+    public $requireSource = false;
+
+    /**
+     * @inheritdoc
+     */
+    public $moduleId = 'comment';
+
+    /**
+     * @var string
+     */
+    public $message;
+
+    /**
+     * @inheritdoc
+     */
+    public function category()
+    {
+        return new CommentNotificationCategory();
+    }
+
+    /**
+     * @inheritdoc
+     */
+    public function html()
+    {
+        return Yii::t('CommentModule.notifications', 'Your comment was deleted by {displayName}. Reason: {message}', [
```

Comment:
I don't think that a name should be given for who deleted a message, rather instead, it should be noted that administration or someone with the ability to delete comments should be displayed, such as a specific user group.

But of course, this also just gets back to the issue for a GroupPicker being developed #5203 for this case.

Reply:
@ArchBlood I would leave the name here for now, then the affected user can contact the person if they have any objections. 

If the name should not be displayed here, a HumHub installation could adjust the translation and remove the `{displayName}`.

Reply:
@s-tyshchenko We definitely still need to save the whole comment in the Notification payload and display an excerpt of the comment in the Notification message. So that the affected users  knows which comment it is.

Reply:
> @ArchBlood I would leave the name here for now, then the affected user can contact the person if they have any objections. 
> 
> If the name should not be displayed here, a HumHub installation could adjust the translation and remove the `{displayName}`.

I can see the need for such case, but it would also invite spam and other negative issues would it not? Personally speaking (for future reference) the user group should be displayed with a way of contacting that user group, which was why I suggested an extended class in the report module. :thinking: